static/js/contextMenu.js

```diff
@@ -0,0 +1,355 @@
+var selectedLineNumber;
+var lines = 0;
+var loginOrEmails = [];
+
+exports.aceSelectionChanged = function(hook, context){
+  var selStart = context.rep.selStart;
+  var selEnd = context.rep.selEnd;
+  if((selStart[0] !== selEnd[0]) || (selStart[1] !== selEnd[1])){
+    iT.show(selStart, selEnd);
+  }else{
+    iT.hide();
+  }
+}
+
+exports.postAceInit = function(hook, context) {
+  
+  attachContextMenu();
+  lines = 0;
+  $('iframe[name="ace_outer"]').contents().find('iframe').contents().find("#innerdocbody").contents().each(function() {
+    lines++;
+  });
+}
+
+function attachContextMenu() {
+  var count = 0;
+  var innerDocBody = $('iframe[name="ace_outer"]').contents().find('iframe').contents().find("#innerdocbody");
+  
+  $(innerDocBody).children().each(function() {
```

Comment:
innerDocBody should already be a jquery object, so you don't need to enclose inside a $() again